migrations/2024-11-06-121933_setup-themes-table/up.sql

```diff
@@ -0,0 +1,17 @@
+-- Your SQL goes here
+CREATE TABLE IF NOT EXISTS themes (
+    theme_id VARCHAR(64) PRIMARY KEY,
+    tenant_id VARCHAR(64) NOT NULL,
+    org_id VARCHAR(64),
+    merchant_id VARCHAR(64),
+    profile_id VARCHAR(64),
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    last_modified_at TIMESTAMP NOT NULL DEFAULT now()
```

Comment:
Let's avoid the defaults on the database side, and have the application always provide it. This would also help prevent issues with the timestamps being generated based on the database server's time zone configuration.